pymchelper/input_output.py

```diff
@@ -20,7 +20,15 @@
 
 
 class AggregationType(IntEnum):
-    """Enum for different types of aggregation."""
+    """
+    Enum for different types of aggregation.
+    Below few examples of such aggregation types used in SHIELD-HIT12A:
+      - NoAggregation is used for density (RHO) and material scorer.
+      - Sum is used for particle counter (COUNT).
```

Comment:
X = sum^j_i x_i          # j = number of jobs

Reply:
Thats correct, for implementation:

https://github.com/DataMedSci/pymchelper/blob/52947dc09b1cc2af172213102541f243ca132095/pymchelper/averaging.py#L208-L219